.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
   - DJANGO_VERSION=1.5
```

Comment:
Yeah, but I don't see any point making our tests more complex to support a version of Django that isn't even getting security patches any more.